tests/Congress.test.ts

```diff
@@ -1697,3 +1726,174 @@ describe("Test for the removing of validators", () => {
         );
     });
 });
+
+describe("Test of Proposal API", () => {
```

Comment:
Proposal tests should be in the `Congress.test.ts`

Reply:
These are in Congress.test.ts but in separate test suit

Reply:
Okay

Reply:
The following error log occurs when testing by executing the following command.
The pre-image of the sample data is not an accurate value.
[Stoa] 2021-11-18T05:46:44.318Z error Verification failed of the previous pre-image and the pre-image.
[Stoa] 2021-11-18T05:46:46.942Z error Verification failed of the previous pre-image and the pre-image.

Reply:
Don't add sample data and use Block Manager.
It will be difficult to generate a pre-image from an already prepared sample.